components/emoji_picker/emoji_list.jsx

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import EmojiListManager from './emoji_list_manager';
+
+const SCROLL_STOP_DELAY = 50;
+const STYLE_WRAPPER = {
+    willChange: 'transform',
+    WebkitOverflowScrolling: 'touch'
```

Comment:
`SCROLL_STOP_DELAY` is the minimum time to determine whether the scrolling stops. The `emoji_list` presents emoji rows on the viewing area only (+ overscan) but during scrolling, it's preventing to do it until it's determined to be on stop status.
I just picked up 50ms but it can be tuned up depending on how we wanted to skip loading emojis. 0ms means no skip.